akka-docs/src/test/java/jdocs/persistence/testkit/PersistenceTestKitPolicySampleTest.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package jdocs.persistence.testkit;
+
+import akka.actor.testkit.typed.javadsl.TestKitJunitResource;
+import akka.actor.typed.ActorRef;
+import akka.persistence.testkit.JournalOperation;
+import akka.persistence.testkit.PersistenceTestKitPlugin;
+import akka.persistence.testkit.ProcessingPolicy;
+import akka.persistence.testkit.ProcessingResult;
+import akka.persistence.testkit.ProcessingSuccess;
+import akka.persistence.testkit.StorageFailure;
+import akka.persistence.testkit.WriteEvents;
+import akka.persistence.testkit.javadsl.PersistenceTestKit;
+import akka.persistence.typed.PersistenceId;
+import com.typesafe.config.ConfigFactory;
+import jdocs.AbstractJavaTest;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+// #policy-test
+public class PersistenceTestKitPolicySampleTest extends AbstractJavaTest {
+
+  @ClassRule
+  public static final TestKitJunitResource testKit =
+      new TestKitJunitResource(
+          PersistenceTestKitPlugin.getInstance()
+              .config()
+              .withFallback(ConfigFactory.defaultApplication()));
+
+  PersistenceTestKit persistenceTestKit = PersistenceTestKit.create(testKit.system());
+
+  @Before
+  public void beforeEach() {
+    persistenceTestKit.clearAll();
+    persistenceTestKit.returnDefaultPolicy();
```

Comment:
Confusing name this, I guess it resets the default policy rather than returns it?

Reply:
hmmm... it is not resetting. Currenlty when you call withPolicy, it reassigns the policy of the testkit. So you call returnDefaultRolicy to assign the default one, back.

Reply:
To me "reset" means exactly that, go back to default.

Reply:
now I got, what you mean :). It is not returning the value :). IMHO no idea of which name is better

Reply:
Probably something like `resetPolicy()` or `resetToDefaultPolicy` is better

Reply:
I was a bit confused, cause "resetting default policy" to me means "reset something (settings) in the default policy" :).

Reply:
So finally, we can rename this. Not sure how you vote for renaming if it's needed I can do that. Should I make an issue?

Reply:
Both those names are fine by me (also sneaking that into this PR would be fine)